pkg/http/http.go

```diff
@@ -173,6 +173,10 @@ func (c *Client) Upload(data UploadRequestData) (*http.Response, error) {
 }
 
 // SendRequest sends an http request with a defined method
+//
+// On error, any Response can be ignored. A non-nil Response with a
+// non-nil error only occurs when CheckRedirect fails, and even then
```

Comment:
There is no `CheckRedirect` in our http package. That `CheckRedirect` refers to one of the http clients which is used internally. Maybe we should clarify that - or omit if not important enough.

I had a look into our coding. I learned there might be an `http.Client` or a `retryablehttp` client.  See [here](https://github.com/SAP/jenkins-library/blob/master/pkg/http/http.go#L234). I don't know if that retryable client behaves like the other client from that the docu above has been copied.

Reply:
I changed the text, please have another look.

I had a quick glance at the client implementation of `retryablehttp` and it looks like the [in error case `nil` is returned for the response.](https://github.com/hashicorp/go-retryablehttp/blob/master/client.go#L693)